patches/minecraft/net/minecraft/client/renderer/BufferBuilder.java.patch

```diff
@@ -58,5 +58,13 @@
 +    public boolean isColorDisabled()
 +    {
 +        return this.field_78939_q;
++    }
++
++    public void addVertexData(ByteBuffer buffer)
```

Comment:
Would suggest `putBulkData`, as that seems to be the "official" name (https://github.com/ModCoderPack/MCPBot-Issues/issues/503).

Reply:
Done